File: JAVA (BONUS)/Nasabah.java
public class Nasabah {
    // Atribut private
    private String idNasabah;
    private String nama;
    private String alamat;

    // Constructor kosong
    public Nasabah() {
    }

    // Constructor
    public Nasabah(String idNasabah, String nama, String alamat) {
        this.idNasabah = idNasabah;
        this.nama = nama;
        this.alamat = alamat;
    }

    // Getter Setter idNasabah
    public String getIdNasabah() {
        return idNasabah;
    }

    public void setIdNasabah(String idNasabah) {
        this.idNasabah = idNasabah;
    }

    // Getter Setter nama
    public String getNama() {
        return nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    // Getter Setter alamat
    public String getAlamat() {
        return alamat;
    }

    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }
}

File: CPP/nasabah.cpp
#include <iostream>
#include <string>

using namespace std;

class Nasabah {
    // Atribut private
    private:
        string idNasabah;
        string nama; 
        string alamat;

    public:
        // Constructor kosong
        Nasabah() {
        }

        // Constructor
        Nasabah(string idNasabah, string nama, string alamat) {
            this->idNasabah = idNasabah;
            this->nama = nama;
            this->alamat = alamat;
        }

        // getter setter id nasabah
        string getIdNasabah() {return idNasabah;}
        
        void setIdNasabah(string idNasabah) {this->idNasabah = idNasabah;}
        
        // getter setter nama
        string getNama() {return nama;}
        
        void setNama(string nama) {this->nama = nama;}
        
        // getter setter alamat
        string getAlamat() {return alamat;}
        
        void SetAlamat(string alamat) {this->alamat = alamat;}
        
    // Destructor
    ~Nasabah() {    
    }
};